resources/views/appeals/appeallist.blade.php

```diff
@@ -45,61 +45,69 @@
-                            <td style="vertical-align: middle;">{{ $appeal['submitted'] }}</td>
-                        </tr>
-                        @endforeach
+                    @foreach($appeals[$type] as $appeal)
+                        @if($appeal['status']=="ADMIN")
+                            <tr class="bg-primary">
+                        @elseif($appeal['status']=="TOOLADMIN")
+                            <tr class="bg-info">
+                        @elseif($appeal['status']=="CHECKUSER")
+                            <tr class="bg-warning" style="color: #212529!important;">
+                        @elseif($appeal['status']=="PROXY")
+                            <tr class="bg-warning" style="color: #212529!important;">
+                        @else
+                            <tr>
+                        @endif
+                                <td style="vertical-align: middle;">
+                                    @isset($appeal['handlingadmin'])
+                                    <a href="/appeal/{{ $appeal['id'] }}" class="btn btn-danger">
+                                    @endisset
+                                    @empty($appeal['handlingadmin'])
+                                    <a href="/appeal/{{ $appeal['id'] }}" class="btn btn-primary">
+                                    @endempty
+                                        #{{ $appeal->id }}
+                                    </a>
+                                </td>
+                                <td style="vertical-align: middle;">{{ $appeal['appealfor'] }}</td>
+                                <td style="vertical-align: middle">
+                                    {{ $appeal->status }}<br/>
+                                    @if($appeal->blocktype === 0)
+                                        IP address
+                                    @elseif($appeal->blocktype === 1)
+                                        Account
+                                    @elseif($appeal->blocktype === 2)
+                                        IP underneath<br /> account
+                                    @else
+                                        Unknown type: {{ $appeal->blocktype }}
+                                    @endif
+                                    on {{ $appeal->wiki }}
+                                </td>
+                                <td style="vertical-align: middle;">{{ $appeal['blockingadmin'] }}</td>
+                                <td style="vertical-align: middle;">{!! $appeal->getFormattedBlockReason('style="color: #00ffea!important;"') !!}</td>
+                                <td style="vertical-align: middle;">{{ $appeal['submitted'] }}</td>
+                            </tr>
+                    @endforeach
+                
                 </tbody>
             </table>
         </div>
     </div>
+    <br />
```

Comment:
Add `mb-4` class to `<div class="card">` instead

Reply:
No clue what line this is referring to. Could you direct suggest? I didn't modify any of the classes in this file.

Reply:
Referring to this line above:
https://github.com/UTRS2/utrs/blob/a53418727fc85e6ca464e264b887bb7ffc590ead/resources/views/appeals/appeallist.blade.php#L49